package.json

```diff
@@ -19,6 +19,7 @@
   },
   "devDependencies": {
     "@biomejs/biome": "^1.9.4",
+    "firebase-tools": "^13.32.0",
```

Comment:
```suggestion
```

Declaring a dependency at the root implies that it can be used across the entire monorepo.